petri/src/vm/openvmm/construct.rs

```diff
@@ -50,7 +52,6 @@ use hvlite_defs::config::Vtl2Config;
 use hvlite_defs::config::DEFAULT_MMIO_GAPS;
 use hvlite_defs::config::DEFAULT_MMIO_GAPS_WITH_VTL2;
 use hvlite_defs::config::DEFAULT_PCAT_BOOT_ORDER;
-use hvlite_helpers::crash_dump::spawn_dump_handler;
```

Comment:
I suspect this was the only other code sharing this method. if so - can you move the method out of helpers, and inline it where its being used?

Reply:
Or maybe make it generic so it can be unified with the new one?